EIPS/eip-7620.md

```diff
@@ -1,28 +1,26 @@
 ---
 eip: 7620
 title: EOF Contract Creation
-description: Introduce `EOFCREATE`, `TXCREATE`, `RETURNCONTRACT` instructions along with a new `InitcodeTransaction` transaction
+description: Introduce `EOFCREATE` and `RETURNCONTRACT` instructions
 author: Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Andrei Maiboroda (@gumb0), Piotr Dobaczewski (@pdobacz)
 discussions-to: https://ethereum-magicians.org/t/eip-7620-eof-contract-creation-instructions/18625
 status: Review
 type: Standards Track
 category: Core
 created: 2024-02-12
-requires: 170, 1559, 2028, 2718, 3540, 3541, 3670, 3860
+requires: 170, 3540, 3541, 3670
 ---
 
 ## Abstract
 
-EVM Object Format (EOF) removes the possibility to create contracts using creation transactions (with an empty `to` field), `CREATE` or `CREATE2` instructions. We introduce three new instructions: `EOFCREATE`, `TXCREATE` and `RETURNCONTRACT`, as well as a new transaction type (`InitcodeTransaction`) to provide a way to create contracts using EOF containers.
+EVM Object Format (EOF) removes the possibility to create contracts using `CREATE` or `CREATE2` instructions. We introduce a new/replacement method in form of pair of instructions : `EOFCREATE` and `RETURNCONTRACT` to provide a way to create contracts using EOF containers.
 
 ## Motivation
     
 This EIP uses terminology from the [EIP-3540](./eip-3540.md) which introduces the EOF format.
 
-EOF aims to remove code observability, which is a prerequisite to legacy EVM contract creation logic using create transactions, `CREATE` or `CREATE2`, because both the initcode and code are available to the EVM and can be manipulated. On the same premise, EOF removes opcodes like `CODECOPY` and `EXTCODECOPY`, introducing EOF subcontainers as a replacement to cater for factory contracts creating other contracts.
+EOF aims to remove code observability, which is a prerequisite to legacy EVM contract creation logic using legacy-style create transactions, `CREATE` or `CREATE2`, because both the initcode and code are available to the EVM and can be manipulated. On the same premise, EOF removes opcodes like `CODECOPY` and `EXTCODECOPY`, introducing EOF subcontainers as a replacement to cater for factory contracts creating other contracts.
     
-The new instructions and the new transaction type introduced in this EIP operate on EOF containers enabling all use cases of contract creation that legacy EVM has.
```

Comment:
This sentence was kinda main part of Motivation, need some replacement, e.g.
```
The new instructions introduced in this EIP operate on EOF containers enabling factory contract use case that legacy EVM has.
```

Reply:
Thanks, I couldn't find a good wording for the replacement somehow.